yaml/xyz/openbmc_project/Logging/AuditLog.interface.yaml

```diff
@@ -0,0 +1,43 @@
+description: >
+    This interface provides methods to extract the Linux audit logged events.
+methods:
+    - name: ParseAuditLog
```

Comment:
If you do end up keeping 2 methods, maybe the names could be more similiar, either both Get* or both Parse*

Reply:
Good point. The Parse name was an artifact of a temporary method I put in for development/debug purposes. I'll rename it.